server/src/types/grades.ts

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+export interface Grade {
+  attainmentId: number,
+  points: number
+}
+
+export interface Student {
```

Comment:
It seems that the point of this interface is to store info about a student and their grades, so its name should reflect that. Maybe `GradesOfStudent`?

Reply:
updated to `StudentGrades`